projects/packages/connection/legacy/class-jetpack-signature.php

```diff
@@ -94,7 +94,7 @@ public function sign_current_request( $override = array() ) {
 			// and encoded on the Jetpack side.
 			if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
 				// phpcs:ignore WordPress.Security.NonceVerification.Missing
-				if ( empty( $body ) && is_array( $_POST ) && count( $_POST ) > 0 ) {
+				if ( empty( $body ) && is_array( $_POST ) && $_POST !== array() ) {
```

Comment:
Quick question about this change, and similar ones in other PRs: here wouldn't it be more readable to check if the array if not empty?

```suggestion
				if ( empty( $body ) && is_array( $_POST ) && ! empty( $_POST ) ) {
```

Reply:
Hi @jeherve!
`rector` rules make these suggestions because of performance reasons: https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#countarraytoemptyarraycomparisonrectorr